apps/backend/src/routes/me/index.ts

```diff
@@ -49,19 +49,24 @@ router.post<UserUpdateRequestBody, ApiResponseBody<UserUpdateResponseBody>>(
   }
 );
 
-router.post<TcxFileUpload, ApiResponseBody<StravaUpload>>(
+const fileUpload = multer().single('file');
+
+router.post<any, ApiResponseBody<StravaUpload>>(
   '/upload',
+  fileUpload,
   async (req, res) => {
-    if (!validationService.authenticateToken(req, res)) {
-      return;
-    }
-    const tcxFile = req.body.tcxFile as string;
-    if (!tcxFile) {
+    if (!req.file) {
       return res.send({
         status: ApiStatus.FAILURE,
-        message: 'BAD REQUEST : MISSING tcxFile',
+        message: 'Bad request : missing file',
```

Comment:
hate the usage of whitespace in front of `:`:))))))))))

I think we should consider simplifying the message en let the status code speak for it self – aka add a `res.status(400)` and change message to `No file provided` or something 🤠

Reply:
Yes, i agree. I just kept going with the unorthodox errors we have used before